agent/engine/docker_image_manager.go

```diff
@@ -345,7 +345,9 @@ func (imageManager *dockerImageManager) deleteImage(imageID string, imageState *
 		return
 	}
 	seelog.Infof("Removing Image: %s", imageID)
-	err := imageManager.client.RemoveImage(imageID, removeImageTimeout)
+	ctx, cancel := context.WithTimeout(context.Background(), removeImageTimeout)
```

Comment:
You shouldn't be using `context.Background()` here. Instead, it should be wired all the way from `performPeriodicImageCleanup()`.

Reply:
Previously, in the RemoveImage function of docker client, it uses context.Background() directly, 
so I use context.Background() in the caller, you want me to do the change?

Reply:
yes please